components/Homepage/AboutUs.js

```diff
@@ -0,0 +1,50 @@
+import { PHASE_PRODUCTION_SERVER } from "next/dist/shared/lib/constants";
+import Button from "../Common/Button";
+import Text from "../Common/Text";
+export default function About(){
```

Comment:
the name of the file should be the same as the component `AboutUs.js` name component should be `AboutUs`